src/components/Education.tsx

```diff
@@ -55,7 +56,7 @@ const Education = () => {
     }, [])
 
     const fetchEducation = async () => {
-        const res = await fetch("http://localhost:8081/education");
+        const res = await fetch(url + "/education");
```

Comment:
@ddinh please store `url + "/education"` to a variable and pass here to minimize code duplication